find-broken-wikilinks.py

```diff
@@ -64,6 +64,15 @@ def file_iterator(roots: list):
             yield item
 
 
+def _identifier_suggestions(e, articles):
```

Comment:
disagree with the leading underscore

```suggestion
def identifier_suggestions(e, articles):
```

Reply:
either works